base/uk.ac.stfc.isis.ibex.ui.scriptgenerator/src/uk/ac/stfc/isis/ibex/ui/scriptgenerator/views/ScriptGeneratorViewModel.java

```diff
@@ -115,6 +119,11 @@
 	 */
 	private static final String GENERATED_SCRIPT_FILEPATH_PROPERTY = "generated script filepath";
 	
+	/**
+	 * The property to listen for changes in Generator containing the generated parameter values data 
+	 */
+	private static final String GENERATED_DATA_EXCHANGE_FILEPATH_PROPERTY = "generated data exchange filepath";
```

Comment:
Shouldn't need to use this as it becomes a getter?

Reply:
Removed.